test/cases/compile_errors/packed_union_fields_mismatch.zig

```diff
@@ -0,0 +1,14 @@
+export fn entry1() void {
+    _ = packed union {
+        a: u1,
+        b: u2,
+    };
+}
+
+// error
+// backend=stage2
+// target=native
```

Comment:
Oh no, not you too! These two values are defaults, I'm trying to get everyone to *not* redundantly specify them in every case :^)